bin/deploy-hobby

```diff
@@ -12,7 +12,7 @@ export SENTRY_DSN="${SENTRY_DSN:-'https://public@sentry.example.com/1'}"
 POSTHOG_SECRET=$(head -c 28 /dev/urandom | sha224sum -b | head -c 56)
 export POSTHOG_SECRET
 
-ENCRYPTION_KEY=$(openssl rand -hex 16)
+ENCRYPTION_SALT_KEYS=$(openssl rand -hex 16)
```

Comment:
ugh, I was so focussed on the upgrade script I didn't build from scratch and totally missed this 🙈 

![was-that-wrong](https://github.com/user-attachments/assets/3ae95de3-a036-4485-9b8c-368887f36b48)